packages/lesswrong/components/editor/EditLinkpostUrl.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useRef } from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import Input from "@material-ui/core/Input";
+import { DatabasePublicSetting } from "../../lib/publicSettings";
+
+const placeholderSetting = new DatabasePublicSetting<string>("linkpostUrlPlaceholder", "http://johnsalvatier.org/blog/2017/reality-has-a-surprising-amount-of-detail")
```

Comment:
Seems a little weird to me that we're actually hard-coding this URL as the default, rather than just using an empty string and adding the URL via the setting.